tests/integration_tests.rs

```diff
@@ -0,0 +1,41 @@
+#![cfg(feature = "integration-tests")]
+#![feature(async_await)]
+
```

Comment:
Could you add unit tests to test the Tso? It really seems like something where we should be able to mock out the pd rpc stuff and test the logic in the Tso itself. (Fine to have an integration test as well, but they don't run without a cluster and tend to test the cluster more than the client itself).

Reply:
Of course, we need a unit test for TSO. An integration test is simper so I make it first.

Reply:
I don't think we must add unit tests for it so early. I've created https://github.com/tikv/client-rust/issues/93 to track it.